qiskit/circuit/library/generalized_gates/rv.py

```diff
@@ -18,7 +18,7 @@
 
 
 class RVGate(Gate):
-    r"""Rotation around arbitrary rotation axis :math:`v` where :math:`|v|` is
+    r"""Rotation around arbitrary rotation axis :math:`\vec{v}` where :math:`|\vec{v}|` is
```

Comment:
While we're at it, could you add the used norm for the vector? Same for the definition below
```suggestion
    r"""Rotation around arbitrary rotation axis :math:`\vec{v}` where :math:`\|\vec{v}\|_2` is
```

Reply:
Done.